src/main/webapp/app/clinical/clinical.component.html

```diff
@@ -61,9 +77,24 @@
                     <ng-select class="selectStyle" [items]="subTypesOptions[clinicalInput.main_type]" [(ngModel)]="clinicalInput.sub_type"
                         [clearable]="true" placeholder="Select Subtype" (change)="onSingleSelected($event)">
                     </ng-select>
+                    <i class="collapse-margin" aria-hidden="true" title="Display optional attributes"
+                       ngClass="{{ displayOptionalFields ? 'fa fa-angle-double-up' : 'fa fa-angle-double-down'}}" (click)="collapse()"></i>
```

Comment:
I don't think the collapse icons to toggle the optional attributes are necessary. Firstly, if you think about your optional concept, all the attributes are optional if I understand things correctly. It doesn't make sense to toggle only these three new attributes. Secondly, there are only one or two attributes, it doesn't hurt to always keep them.